package.json

```diff
@@ -9,12 +9,13 @@
   "author": "redpencil.io",
   "license": "MIT",
   "dependencies": {
-    "@lblod/mu-auth-sudo": "^0.2.0",
     "@lblod/marawa": "^0.7.1",
+    "@lblod/mu-auth-sudo": "^0.2.0",
     "body-parser": "1.19.0",
     "fs-extra": "8.1.0",
     "jsdom": "^11.6.2",
-    "lodash": "^4.17.11"
+    "lodash": "^4.17.11",
+    "uuid": "^8.3.2"
```

Comment:
is uuid not in mu-auth-sudo?

Reply:
The uuid exposed by the mu-javascript-template is uuidv1 (see https://github.com/mu-semtech/mu-javascript-template/blob/master/helpers/mu/index.js#L6) which is based on timestamp and not random. So I was ending up with the same uuids instead of unique ones